ChangeLog.md

```diff
@@ -2,8 +2,9 @@
 
 ## Unreleased
 
-- `Commutative (Product a)` now merely requires
-  `Num a`.
+- `Commutative (Product a)` now merely requires `CommutativeProduct a`.
```

Comment:
```suggestion
- `Commutative (Product a)` now requires `CommutativeProduct a`.
```
Perhaps also mention some things we found in our discussion?

Reply:
Yeah I just did the bare minimum so the change log wouldn't be wrong. Feel free to propose what it ought to say from scratch if you like.

Reply:
Marked as draft because there are many more Num instances in base to add.

Reply:
#5 fleshes out the changelog entry, and is set to merge into this branch.